src/Archives/VolFile.h

```diff
@@ -64,13 +66,15 @@ namespace Archives
 		{
 			SectionHeader();
 			SectionHeader(std::array<char, 4> tag, uint32_t length, VolPadding padding = VolPadding::FourByte);
+
 			std::array<char, 4> tag;
 			uint32_t length : 31;
 			VolPadding padding : 1;
 		};
-#pragma pack(pop)
 
-		static_assert(sizeof(SectionHeader) == 8, "SectionHeader not of required size");
+		static_assert(8 == sizeof(SectionHeader), "VolFile::SectionHeader is an unexpected size");
```

Comment:
I approve of the order swap here. That reads better with the hardcoded size on the left, and matches typical assert order.